src/ecs/resources/input/input_handler.rs

```diff
@@ -260,7 +285,7 @@ impl InputHandler {
     }
 
     /// Returns the value of an axis by the string id, if the id doesn't exist this returns None.
-    pub fn axis_value<T: AsRef<str>>(&self, id: T) -> Option<f32> {
+    pub fn axis_value<T: AsRef<str>>(&self, id: T) -> Option<f64> {
```

Comment:
we don't gain any value from using an f64 here, it's just more empty bits as this function can only return -1, 0, and 1.